ci_scripts/check_fauxpas.sh

```diff
@@ -19,4 +19,5 @@ fi
 
 echo "Linting with Faux Pas..."
 
-fauxpas check Stripe.xcodeproj/ --target "StripeiOSStatic" --configFile "./FauxPasConfig/main.fauxpas.json" --minErrorStatusSeverity Concern
+set -e fauxpas check Stripe.xcodeproj/ --target "StripeiOSStatic" --configFile "./FauxPasConfig/main.fauxpas.json" --minErrorStatusSeverity Concern
```

Comment:
You can just `set -e` once at the top of the file; you don't have to prepend it to every line.